src/Ojs/SiteBundle/Resources/views/Search/index.html.twig

```diff
@@ -130,6 +130,9 @@
                                                 {% set url = resultData.detail.route %}
                                                 {% set journal = journal|merge({'title':resultData.detail.name}) %}
                                                 {% include 'journal.html.twig' %}
+                                            {% elseif key == 'publisher' %}
```

Comment:
i think this logic is not fine we must to generate link in SearchManager 

related link;
https://github.com/ojs/ojs/blob/master/src/Ojs/CoreBundle/Service/Search/SearchManager.php#L201